cmd/dynamic_startup.go

```diff
@@ -130,20 +123,70 @@ func ValidateDynamicStartupFlags(accessPublicKey, accessAddress string, startPha
 		return fmt.Errorf("invalid flag --dynamic-startup-startup-epoch-phase unknow epoch phase")
 	}
 
-	if startEpoch <= 0 {
-		return fmt.Errorf("invalid flag --dynamic-startup-startup-epoch must be integer > 0")
+	return nil
+}
+
+// DynamicStartPreInit will attempt to save a the latest finalizes protocol snapshot
+// if we have reached the target epoch and phase are
```

Comment:
We can probably workshop this a bit 😛

Reply:
https://github.com/onflow/flow-go/pull/1893/commits/a24cc2a5241a68045645e4d1203580e1f759b6ae